prime_numbers.py

```diff
@@ -0,0 +1,31 @@
+def generate_primes(n=100):
+    """
+    This function generates all prime numbers between 1 to n.
```

Comment:
Consider adding type hints to the function parameters for better readability and self-documenting code.